src/Mod/Draft/draftfunctions/upgrade.py

```diff
@@ -227,7 +227,7 @@ def makeShell(objectslist):
                 # of downgrade, nor do they have the same hashCode().
                 # Nevertheless, they still keep reference to their original
                 # colors, capture that in facecolors.
```

Comment:
ShapeAppearance has replaced ShapeColor. This reversion, although just in a comment, is incorrect.